spec/Provider/TaxRateProviderSpec.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\InvoicingPlugin\Provider;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\AdjustmentInterface;
+use Sylius\Component\Core\Model\OrderItemUnitInterface;
+use Sylius\InvoicingPlugin\Exception\MoreThanOneTaxAdjustment;
+use Sylius\InvoicingPlugin\Provider\TaxRateProviderInterface;
+
+final class TaxRateProviderSpec extends ObjectBehavior
+{
+	public function it_implements_tax_rate_provider_interface(): void
+	{
+		$this->shouldImplement(TaxRateProviderInterface::class);
+	}
```

Comment:
You are using wrong indentation in all your PHP files 😢

Reply:
I don't know why, in my phpstorm window looks normal and seetings are like on other projects 
<img width="788" alt="Screenshot 2021-07-22 at 07 17 53" src="https://user-images.githubusercontent.com/39232096/126593021-ceb8a368-66be-4195-b6f1-260b21477166.png">